package.json

```
@@ -91,12 +91,10 @@
     "npm-run-all": "^4.1.5"
   },
   "devDependencies": {
+    "@1024pix/eslint-config": "^1.0.0",
```

Compte tenu que l'organisation 1024pix est maîtresse de ses dépôts Git, pourquoi passer par la publication du paquet *@1024pix/eslint-config* sur NPM plutôt qu'utiliser ce paquet directement depuis son dépôt GitHub cf. https://docs.npmjs.com/cli/v8/configuring-npm/package-json#github-urls ?

```suggestion
    "@1024pix/eslint-config": "1024pix/eslint-config#v1.0.0",
```

Cette remarque vaut également pour Pix-UI.

---

Intéressant, je ne connaissais pas cette possibilité :) à discuter !

---

Encore plus direct: récupérer la dernière version (pas au sens release, mais au sens git)
On le fait souvent dans l'infra quand on veut aller rapidemenent en production, ex. pour oppsy
https://github.com/1024pix/pix/blob/dev/api/package.json#L61C54-L61C54

Ca donnerait ici
```suggestion
    "@1024pix/eslint-config": "https://github.com/1024pix/eslint-config#main",
```

---

C'est vrai uniquement dans le cas où tu n'as pas de package-lock @octo-topi non ? 🤔

---

Aucune idée. `oppsy` a bien un `package-lock.json` https://github.com/1024pix/oppsy

---

Je parlais du côté de là où il est installé https://github.com/1024pix/pix/blob/dev/api/package-lock.json#L9093C34-L9093C103, on voit bien que même en disant `#main` si on souhaite mettre à jour la lib il faut faire un `npm i`

---

Je confirme que la version de oppsy est sauvegardé dans le package-lock, on ne vas chercher la dernière version a chaque installation des dépendances. Et le package-lock du dépot n'est jamais utilisé.

---

:exploding_head:

---

Par curiosité, ces propositions sont elles compatibles avec Renovate ? Je n'arrive pas à trouver de réponse simple à cette question

---

Je pense que oui, quand renovate met à jour le lock file, il doit surement faire un `npm i` et commiter le package-lock

---

Je laisse la conversation ouverte pour l'historique